SLE/wizard/installation.qss

```diff
@@ -1,4 +1,11 @@
+/* See the Qt style sheet reference documentation at */
+/* http://doc.qt.io/qt-5/stylesheet-reference.html for the list of supported */
+/* properties and allowed values. */
+
 /* Richtext: installation_richtext.css */
+/* Do not touch this comment! It is parsed by libyui-qt when loading the style! */
```

Comment:
It's a comment for a comment :) But comments are expected to be above the commented item. Could you PLS move it above then?